### PR TITLE
Bug 2102743: RBAC permissions are not granted for ServiceMonitor CR

### PR DESCRIPTION
**Pull request by rauhersu**
Permissons added for the monitoring group when making requests
to the API server for the actions specified under role.yaml

### CODE REVIEW

**Comment by jc-rh** [comment]
Or raise a new bug targetting 4.12 as this should also be backported?

**Comment by rauhersu** [comment]
> Or raise a new bug targetting 4.12 as this should also be backported?

Thanks for clarifying @jc-rh , new BZ created

**Comment by rauhersu** [comment]
pinging @jc-rh and @donpenney for lgtm

File: config/rbac/role.yaml [hunk 1]
```diff
@@ -135,3 +135,15 @@ rules:
...
+  resources:
+  - servicemonitors
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```
Comment (donpenney) [inline config/rbac/role.yaml:149]: nit: Please ensure the file ends with a new-line character

**Review by donpenney — APPROVED** [review]
/lgtm

**Pull request merged**